File: WorkIt-Server/WorkIt-Server/Models/ViewModels/DialogsListViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WorkIt_Server.Models.ViewModels
{
    public class DialogsListViewModel
    {
        public int DialogId { get; set; }
        public string User1Picture { get; set; }
        public int User1Id { get; set; }
        public string User1Name { get; set; }
        public string User2Name { get; set; }
        public int User2Id { get; set; }
        public string User2Picture { get; set; }
        public string LastMessageText { get; set; }
        public DateTime? LastMessageCreatedAt { get; set; }
    }
}

File: WorkIt/app/src/main/java/prot3ct/workit/view_models/DialogsListViewModel.kt
package prot3ct.workit.view_models

data class DialogsListViewModel(
    val dialogId: Int,
    val user1Id: Int,
    val user1Name: String,
    val user2Name: String,
    val user2Id: Int,
    val user1Picture: String,
    val user2Picture: String,
    val lastMessageText: String,
    val lastMessageCreatedAt: String
)